Adapter.C

```diff
@@ -110,6 +110,13 @@ bool preciceAdapter::Adapter::configFileRead()
                     }
                     DEBUG(adapterInfo("    connectivity : " + std::to_string(interfaceConfig.meshConnectivity)));
 
+                    // When restarting FSI simulations, we may need to account for previous displacement.
+                    // We do this by resetting the displacement when defining the interface.
+                    // Since this is a feature that may not work as expected, depending on the implementation of the
+                    // structure solver used, we allow the user to enable this feature.
+                    interfaceConfig.resetDisplacement = interfaceDict.lookupOrDefault<bool>("resetDisplacement", false);
```

Comment:
A comment that I had previously made outside a threaded conversation:

I am also wondering if `reset` is the right term here. I always think of "reset" as "going back to the default/zero state", while here we want to "continue the game from the saved point".

Maybe `restart`, `recover`, or something else would fit better?

Reply:
> going back to the default/zero state

Thinking in terms of the interface deformation, that's exactly what's happening here. I would maybe try to find a name that considers the 'restart' association.

Reply:
Renamed it to 'restoreUndeformedInterface', please comment again if you want it to be changed once more,

Reply:
I think that `restoreUndeformedInterface` is similarly confusing as `resetDisplacement`.

Since we want to communicate that this is (only) relevant for restarts, how about something like `restartFromDeformed`?

From the perspective of preCICE, it makes sense that we tell preCICE "these are the locations of the nodes that the simulation originally started from".

From the perspective of the user, however, the important information is the restarting part. To restart from a deformed case, we need to restore the original frame of reference.

Could we even make this a keyword-based option? Something like:
```c++
FSI{
    restartFrom zeroDisplacement;
}
```
with alternative:
```c++
FSI{
    restartFrom lastDisplacement;
}
```

We can then internally map it to the current `resetDisplacement` or `restoreUndeformedInterface`.